src/rpc/rawtransaction.cpp

```diff
@@ -811,7 +811,7 @@ UniValue SignTransaction(CMutableTransaction& mtx, const UniValue& prevTxsUnival
                 }
                 Coin newcoin;
                 newcoin.out.scriptPubKey = scriptPubKey;
-                newcoin.out.nValue = 0;
+                newcoin.out.nValue = MAX_MONEY;
```

Comment:
I was wondering if `MAX_MONEY` should be used, because `MAX_MONEY` (like the value `0` that it replaces) is a valid amount isn't it?